cmd/agent/app/flags.go

```diff
@@ -56,6 +60,22 @@ func AddFlags(flags *flag.FlagSet) {
 		"",
 		"comma-separated string representing host:ports of a static list of collectors to connect to directly (e.g. when not using service discovery)")
 	flags.String(
+		scheme,
+		"http",
+		"protocol scheme to use when talking to the collector")
```

Comment:
What is the scheme for the current protocol (agent-collector)? Can be made that default?

Reply:
Good question. If TChannel is over HTTP (I don't think it is), then it's certainly `http` :) In any case, this is either `http` or `https`, as it's used only for the new reporter and will be ignored when the TChannel reporter is used.

Reply:
I am asking because this flag adds more confusion

Reply:
Added a clarification at the end:

> used only when auth-related properties are specified

Hopefully this clears up the confusion.